### PR TITLE
place pulpcore-worker in libexec

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/python-pulpcore/python-pulpcore.spec [hunk 1]
```diff
@@ -166,17 +166,21 @@ do
   install -D -m 755 ${wrapper} %{buildroot}%{_libexecdir}/%{pypi_name}/${wrapper}
 done
 
+mv %{buildroot}%{_bindir}/pulpcore-worker %{buildroot}%{_libexecdir}/%{pypi_name}/
```
Comment (ekohl) [inline packages/python-pulpcore/python-pulpcore.spec]: Should we use `install -D -m 755` to ensure permissions?

**Comment by evgeni** [comment]
needs to wait for https://pulp.plan.io/issues/8906 to be decided

File: packages/python-pulpcore/python-pulpcore.spec (hunk 1 above)
Comment (evgeni) [reply packages/python-pulpcore/python-pulpcore.spec]: it is in bindir, so I would expect it to already have correct perms.

Comment (evgeni) [reply packages/python-pulpcore/python-pulpcore.spec]: I went with wrapper for consistency

**Review by ekohl — APPROVED** [review]

**Pull request merged**